src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>API: Leaderboard</title>
+</head>
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+
+  <main class="container">
+    <section class="scores-section">
+      <div class="recent">
+        <div class="sub-header">Recent scores</div>
+        <div>
+          <button class="btn refresh-btn">Refresh</button>
+        </div>
+      </div>
+
+      <div class="scores-list">
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+        <div>Lorem: 100</div>
+      </div>
+    </section>
+
+    <section class="input-section">
+      <div class="sub-header">Add your scores</div>
+
+      <form>
+        <input type="text" placeholder="Your name">
+        <input type="text" placeholder="Your score">
+        <div>
+          <button class="btn submit-btn" type="button">Submit</button>
+        </div>
+      </form>
+    </section>
+  </main>
```

Comment:
- [ ] I couldn't be able to see that the application match the given template. This is because I couldn't run the `npm start` command. Please, make the changes to run the command and then, add the HTML code inside the js script in place index.html, this because on this part you'll work with js, at this point to add HTML code is easy for you, but js will help to have a more helpful template to add elements and other important concepts in your code using js. 🌵 
![image](https://user-images.githubusercontent.com/20563252/131336719-a9694d74-f53b-4a03-a380-bc89d9bab948.png)